pkg/sloop/store/typed/tables.go

```diff
@@ -20,6 +20,7 @@ type Tables interface {
 	WatchActivityTable() *WatchActivityTable
 	Db() badgerwrap.DB
 	GetMinAndMaxPartition() (bool, string, string, error)
+	GetMinPartition(txn badgerwrap.Txn) (bool, string, error)
```

Comment:
Its a little inconsistent to have GetMinPartition(txn) and GetMinAndMaxPartitions(no txn), and also to not have a GetMaxParititon(?).  Would it cause problems to have all 3 and have them take in a txn?